.github/CODEOWNERS

```diff
@@ -185,6 +185,7 @@
 /sdk/formrecognizer/                                               @kinelski @pallavit @joseharriaga
 
 # ServiceLabel: %Cognitive - Form Recognizer %Service Attention
+/sdk/documentintelligence/                                         @toothache
```

Comment:
Be aware that this change replaces the existing contacts because the comment applies only to the entry directly below it.   This means that you will be the support contact for anything using `Cognitive - Form Recognizer` when it is marked for service attention.

Reply:
@toothache is that the intention here?

Reply:
It's not the intention. I'll revert the change here.